### ISSUE TITLE
Add ability to change ui-wide font for accessibility

### ISSUE THREAD

**Issue by chaoticryptidz** [feditext/feditext#216]
I wish feditext supported setting a font for the whole UI or just chat messages. Ice Cubes for iOS also lets you do this with a selector that lets you pick between any system font or any font installed via app profiles. Can use this to set font to comic sans, dyslexie, opendyslexic, etc to make it easier to read.

**Describe the solution you'd like**
Add a preference for using any system/installed font with picker that lets you see the font and font name of all fonts in system. preferably it would have sensible system defaults for fallback characters

ice cubes lets you set a font but doesn't change fallbacks to system font so lots of characters and languages end up looking weird or out of place, i don't know if this is a swiftui problem but this could be a potential limitation and should maybe be listed as a warning for the feature such as "[Warning] Some languages and symbols may render weirdly when using a custom font"

**Comment by joan7770** [comment]
I would also like this feature to come about. I love for OpenDyslexic to be bundled as one of the font choices.

**Comment by mirabilos** [comment]
Yes, please! I just managed to get the font family I like on the device with the Fontcase open source äpp, now I need FediText to use it for post display so I can finally figure out whether something is an I, l or 1 again (on the laptop I use a Firefox `userContent.css` to do the same to Semaphore).

Best make two dropdowns, one for regular fonts and one for monospaced fonts (I installed that one as well) for ```` ``` … ``` ```` fenced code blocks, `` `…` `` inline code, etc.
